src/Simulation/Simulators/QuantumSimulator/QubitManager.cs

```diff
@@ -64,12 +64,14 @@ protected override void ReleaseOneQubit(Qubit qubit, bool usedOnlyForBorrowing)
                 if (qubit != null)
                 {
                     bool areAllReleasedQubitsZero = ReleaseOne(this.SimulatorId, (uint)qubit.Id);
-                    if (!areAllReleasedQubitsZero && throwOnReleasingQubitsNotInZeroState)
+                    bool isQubitMeasured = qubit.IsMeasured;
```

Comment:
This seems like a rather unnecessary definition of a variable to me, given that qubit.IsMeasured is as readable and also not longer than the variable name.

Reply:
Yes, will incorporate this!